README.md

```diff
@@ -1,13 +1,15 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
+This is a project I made during the Technigo bootcamp.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was about building a survey form. The purpose of this project was to learn  how to use the useState hook in react and how to connect HTML form inputs to state.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started by creating a board in Trello to plan the project. After that I created the components I wanted to have in my project. I started to create them in codesandbox and the reason for that was so I could practise a bit first and see that I had understood everything about components and how they work. When I started the project in Visual Studio I had a lot of problems with node_modules so my biggest challenge whit this project was to follow my timeplan. I had some difficulties with passing the right information down to my components through props but I searched a lot on github and stackOverflow to see other peoples solutions and that helped me a lot. I did 2 validation tests: validator.w3.org and color contrast accessibility validator. The site passed both of them. I also added tabindex for screenreaders. If I had more time I would create another button in the summary that will take you back to the form again. I would also style the radio buttons and the text inputs in a nicer way. 
+
+
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://book-survey.netlify.app/
```

Comment:
This is also an option if you want to use markdown to hide the link: [I'm an inline-style link](https://www.google.com)